assets/components/src/with-wizard/index.js

```diff
@@ -250,7 +250,7 @@ export default function withWizard( WrappedComponent, requiredPlugins ) {
 		 * Render.
 		 */
 		render() {
-			const { suppressFooter } = this.props;
+			const { footerSimple, footerSuppress } = this.props;
```

Comment:
is `footerSuppress` actually used anywhere? I couldn't find an instance of it being set, so I believe it will always be `false` and therefore can be removed from here and line 279, right?

Reply:
I kept it in case we were going to use it some day but it doesn't make sense to keep it (footer should always be present -- simple or not)
Removed in https://github.com/Automattic/newspack-plugin/pull/885/commits/972ed944203e8f4804ac2dffded3fa4a2c93d363